README.md

```diff
@@ -122,6 +141,7 @@ game.specialDiceRoller.v5
 game.specialDiceRoller.genesys
 game.specialDiceRoller.starWars
 game.specialDiceRoller.heroQuest
+game.specialDiceRoller.HeXXen
```

Comment:
should be lowercase **h**

Reply:
changed. sorry. not really the biggest programmer here.